### PR TITLE
Adding wording which explains the difference between the same context and identical contexts.

### PR DESCRIPTION
**Pull request by dan-tripp-siteimprove**
For rule fd3a94 (Links with identical accessible names and same context serve equivalent purpose): Adding wording which explains the difference between the same context and identical contexts.

Closes issue(s):

- none

Need for Call for Review:
This can be merged with 1 approval << choose reason: editorial changes to website/test code, adding new contributor, other (explain). 
** This PR only changes wording, and (I hope) adds clarity.  It doesn't change any meaning.
---

### **When creating PR:**

- [ ] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [ ] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [ ] Add yourself (and co-authors) as "Assignees" for PR.
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [ ] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by WilcoFiers — APPROVED** [review]

**Review by Jym77 — APPROVED** [review]

File: _rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ There are no accessibility support issues known.
...
 
-This rule specifically targets links within the exact same context. Links with identical name that are in identical (but not the same) contexts also fail [2.4.4 Link Purpose (In Context)][sc244]. However, defining "identical context" unambiguously was not really possible and was left out of this rule.
+There is a difference between two contexts being the *same* and being *identical*.  This rule specifically targets links within the *same* context.  The same context means exactly the same set of DOM nodes.  Identical (but not the same) contexts might have a different set of DOM nodes, but those DOM nodes have equivalent content - such as text content, attribute values, and so on.  This difference is similar to the difference in some programming languages between pointer equivalence and deep object equivalence.  Links with identical name that are in identical (but not the same) contexts also fail [2.4.4 Link Purpose (In Context)][sc244]. However, defining "identical context" unambiguously has been deemed infeasible at this time, and so has been left out of this rule.
```
Comment (carlosapaduarte) [inline _rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md]:
```suggestion
There is a difference between two contexts being the *same* and being *identical*. This rule specifically targets links within the *same* context. The same context means exactly the same set of DOM nodes. Identical (but not the same) contexts might have a different set of DOM nodes, but those DOM nodes have equivalent content - such as text content, attribute values, and so on. This difference is similar to the difference in some programming languages between pointer equivalence and deep object equivalence. Links with identical name that are in identical (but not the same) contexts also fail [2.4.4 Link Purpose (In Context)][sc244]. However, defining "identical context" unambiguously has been deemed infeasible at this time, and so has been left out of this rule.
```

Comment (carlosapaduarte) [inline _rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md]:
```suggestion
There is a difference between two contexts being the *same* and being *identical*.  This rule specifically targets links within the *same* context.  The same context means exactly the same set of DOM nodes.  Identical (but not the same) contexts might have a different set of DOM nodes, but those DOM nodes have equivalent content - such as text content, attribute values, and so on.  Links with identical name that are in identical (but not the same) contexts also fail [2.4.4 Link Purpose (In Context)][sc244]. However, defining "identical context" unambiguously has been deemed infeasible at this time, and so has been left out of this rule.
```

I don't think we need this sentence to explain the different between same and equivalent. But I won't block accepting the PR if you prefer to keep it.

File: _rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md [hunk 2]
```diff
@@ -374,7 +374,7 @@ These two `span` elements do not have a [semantic role][] of link.
 
 #### Inapplicable Example 5
 
-These two HTML `a` elements have the same [accessible name][] and link to the [same resource][] but different [programmatically determined link contexts][programmatically determined link context]. Even if the contexts have identical content, they are not the same (the same elements).
+These two HTML `a` elements have the same [accessible name][] and link to the [same resource][] but different [programmatically determined link contexts][programmatically determined link context]. Even though the two contexts in this example are *identical*, they are not the *same*.  That is: even though they have equivalent content, they do not consist of the same DOM elements.
```
Comment (carlosapaduarte) [inline _rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md]:
```suggestion
These two HTML `a` elements have the same [accessible name][] and link to the [same resource][] but different [programmatically determined link contexts][programmatically determined link context]. Even though the two contexts in this example are *identical*, they are not the *same*. That is: even though they have equivalent content, they do not consist of the same DOM elements.
```

**Review by carlosapaduarte — CHANGES_REQUESTED** [review]
Clearing up whitespace and suggest the removal of a sentence

**Comment by dan-tripp-siteimprove** [comment]
> Clearing up whitespace and suggest the removal of a sentence

Your edits look good to me.  Thank you, Carlos.

**Comment by carlosapaduarte** [comment]
@dan-tripp-siteimprove A quick follow-up on the use of GitHub for the reviewing process (this is something that happens often). Even tough you committed changes I proposed, you will notice that my review is still marked as "changes requested". After you address a review, you need to go to the widget at the end of the page (above the comments box) and expand the "1 change requested" details element. There you will find my review with an ellipsis symbol on the right. Press that and select "Dismiss review". Then you need to go to the top right side bar, and under reviews, re-request a review from me again.
Only after these steps, will GitHub allow me review again and Accept (or not, in case I do not agree with the way my review has been handled) your PR.

**Review by carlosapaduarte — APPROVED** [review]

**Comment by carlosapaduarte** [comment]
> @dan-tripp-siteimprove A quick follow-up on the use of GitHub for the reviewing process (this is something that happens often). Even tough you committed changes I proposed, you will notice that my review is still marked as "changes requested". After you address a review, you need to go to the widget at the end of the page (above the comments box) and expand the "1 change requested" details element. There you will find my review with an ellipsis symbol on the right. Press that and select "Dismiss review". Then you need to go to the top right side bar, and under reviews, re-request a review from me again.
> Only after these steps, will GitHub allow me review again and Accept (or not, in case I do not agree with the way my review has been handled) your PR.

Apparently, my input was wrong :-) Even with the request for changes you were able to request my review, and I was able to complete it!

**Comment by dan-tripp-siteimprove** [comment]
I accidentally clicked something yesterday - one of those "review" buttons, I forget which one. That might have caused some confusion.  At any rate - thank you.

**Pull request merged**